cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -200,7 +200,9 @@
             {% endfor %}
         {% endif %}
         {% if 'filterable-list' in controls.form_type and posts is defined %}
-            {% if posts.paginator.count == 0 %}
+            {% if not request.GET %}
```

Comment:
ok, should we pass in a blank message since you will never see that notification unless an error occurs?